NEWS.md

```diff
@@ -33,6 +33,40 @@
   to install additional packages.
 - Implemented `CacheOnly=metadata`. Note that in the JSON output, the
   value of `CacheOnly=` will now be a string instead of a boolean.
+- Added `CompressLevel=` to set the compression level to use.
+- Dropped experimental Gentoo support.
+- Added `TriggerMatch=` to specify multiple match sections of which only
+  one should be satisfied.
+- Added various packages to the default tools tree.
```

Comment:
I think it would be clearer to enumerate them. If I didn't miss anything looking at it it was
- jq
- attr
- acl
- git
- sed
- grep
- findutils